api/src/main/resources/_core/database/mysql/xf_system/etl_error/sp_mamba_elt_test_log.sql

```diff
@@ -0,0 +1,11 @@
+-- $BEGIN
+CREATE TABLE IF NOT EXISTS mamba_etl_test_log
```

Comment:
@GOmare 
1. is this table recreated every time mamba is deployed?
2. when do the tests run?